media/test_imports/faq_bad_import_labels.csv

```diff
@@ -0,0 +1,4 @@
+Parent ID,Parent Language,Parent Question,Parent Answer,Labels,afr ID,afr Question,afr Answer,bla ID,bla Question,bla Answer
```

Comment:
I believe it is reserved for future use - specifically for allowing FAQ instances to be updated from the CSV upload.